package.json

```diff
@@ -39,6 +39,6 @@
   },
   "dependencies": {},
   "engines": {
-    "nrfconnect": "2.x"
+    "nrfconnect": "2.4.x"
```

Comment:
Should it be `^2.4.0`? I think it is safe to assume that the app will be supported on v2.4 and any minor version above. We are using semver, so no breaking changes should be introduced in core before v3.0.